test.js

```diff
@@ -51,6 +51,56 @@ Ponies|
 	t.is(actual, expected);
 });
 
+test('main2', t => {
+	const actual = bitbar.create([
```

Comment:
This test has too much boilerplate. You only need to test with one item having an URL with a quote.

Reply:
Removed unnecessary codes.